src/hotspot/share/gc/shenandoah/shenandoahFreeSet.cpp

```diff
@@ -74,31 +412,41 @@ HeapWord* ShenandoahFreeSet::allocate_single(ShenandoahAllocRequest& req, bool&
   // Free set maintains mutator and collector views, and normally they allocate in their views only,
   // unless we special cases for stealing and mixed allocations.
```

Comment:
I'd drop this comment, unless such special cases actually exist in the code today. Do they?

Reply:
Thanks for calling this out.  I've refined these comments to reflect current reality.